infra/modules/azuread-grant-all-users/main.tf

```diff
@@ -44,6 +39,16 @@ ${join("\n", concat(var.app_roles, var.oauth2_permission_scopes))}
 EOT
 }
 
+resource "local_file" "todo" {
```

Comment:
In past I had this automated; it seems now can be done through [az cli](https://learn.microsoft.com/en-us/cli/azure/ad/app/permission?view=azure-cli-latest#az-ad-app-permission-admin-consent) and since https://github.com/Worklytics/psoxy/pull/261 we have roles required as part of documentation , so we could get rid of this *todo* and just have the command executed as part of terraform flow.
If worth it, please create a task and assign it to me!